src/boot/chid.h

```diff
@@ -11,18 +11,21 @@
 
 enum {
         DEVICE_TYPE_DEVICETREE = 0x1, /* A devicetree blob */
+        DEVICE_TYPE_FW         = 0x2, /* A firmware blob */
```

Comment:
i'd probably rename this to DEVICE_TYPE_UEFI_FW or so, i.e. make this just about uefi for now. if we later on add support for other types of fw, we can then add more types like this. otherwise we have to add a secondary fw type field somewhere, and I think it would be simpler to just have a single field which indicates both that this is a firmware, and what kind of firmware it is.